app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file_name = command.split()[1]
+    new_file_name = command.split()[2]
```

Comment:
don't split one object twice - it's redundant work of CPU
```suggestion
    splitted_command = command.split()
    old_file_name = splitted_command[1]
    new_file_name = splitted_command[2]
```